### ISSUE TITLE
Staff Create Pet: add tooltips to the published and paused applications toggles

### ISSUE THREAD

**Issue by kasugaijin** [rubyforgood/homeward-tails#1220]
These two toggles might not be intuitive to understand what they are for, so it would be good to add an icon and tooltip description  for each. See an example of a tooktip in action on our theme here (for password strength): https://geeksui.codescandy.com/geeks/pages/security.html 

Application Paused: 'If applications are paused, the public can see the pet but cannot submit an adoption application.'
Published: 'If a pet is published it is visible to the public.'

<img width="380" alt="image" src="https://github.com/user-attachments/assets/a82acdc2-ba37-4717-bae9-e9e83284182c">

**Comment by mgrigoriev8109** [comment]
Can I work on this one @kasugaijin ?

**Comment by kasugaijin** [comment]
For sure all yours @mgrigoriev8109

**Comment by mgrigoriev8109** [comment]
Unassigning myself from this if anyone else wants to work on it - 

assuming no one else has implemented actual responsive tooltips yet, I think the temporary fix we want to have is to include those two messages in parenthesis next to the Switch buttons.

**Comment by edwinthinks** [comment]
@kasugaijin I think actually you can just change the labels to avoid the tooltip.

"Application Paused" -> "Accept new adoption applications"
"Published" -> "Pet visible to public?"

Sometimes I think we can add a bit more details in the toggle to make it more clear rather than go for the tooltip where you'd have to dive into the tooltip to understand.

**Comment by kasugaijin** [comment]
@edwinthinks yes we went with modified labels a s a workaround in the meantime. The tooltip is definitely a nice to have, and you're right, to be used cautiously.

**Comment by edwinthinks** [comment]
@kasugaijin yeh tooltips is good but I don't often think people use them to learn more about anything. Would be a good additional touch if you find people don't know what those mean.
